services/github/github-release-date.service.js

```diff
@@ -8,21 +8,28 @@ import { documentation, errorMessagesFor } from './github-helpers.js'
 const schema = Joi.alternatives(
   Joi.object({
     created_at: Joi.date().required(),
+    published_at: Joi.date().required(),
   }).required(),
   Joi.array()
     .items(
       Joi.object({
         created_at: Joi.date().required(),
+        published_at: Joi.date().required(),
       }).required()
     )
     .min(1)
 )
 
+const queryParamSchema = Joi.object({
+  published_at: Joi.equal(''),
```

Comment:
Rather than a flag, I think the way we should present this is a `display_date` parameter with 2 possible string values: `created_at` and `published_at`. The default is `created_at`. The GitLab releases badge has some good examples of this:
https://github.com/badges/shields/blob/523e31d3893abba6f6ecd9e16c20ff089d4e5d98/services/gitlab/gitlab-release.service.js#L15-L23